src/index.html

```diff
@@ -7,6 +7,42 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main>
+      <div
+      class="calendar calendar--month-length-29 calendar--weekday-sun"
+      >
```

Comment:
```suggestion
      <div class="calendar calendar--month-length--29 calendar--weekday--sun">
```

Reply:
lint failed in your variant because of string length

Reply:
```suggestion
      <div
        class="calendar calendar--month-length--29 calendar--weekday--sun"
      >
```
use "--" with modifier value, p.2

Reply:
done